pkg/liveshare/rpc.go

```diff
@@ -40,7 +42,60 @@ func (r *rpcClient) do(ctx context.Context, method string, args, result interfac
 	return waiter.Wait(waitCtx, result)
 }
 
-type nullHandler struct{}
+type handlerFn func(conn *jsonrpc2.Conn, req *jsonrpc2.Request)
 
-func (nullHandler) Handle(ctx context.Context, conn *jsonrpc2.Conn, req *jsonrpc2.Request) {
+type requestHandler struct {
```

Comment:
Combine this with rpcClient.

Reply:
Do you mean get rid of requestHandler? I think we still need a struct here to implement Handle so we can create the connection in line 26

Reply:
done